core/src/main/java/jenkins/model/Nodes.java

```diff
@@ -256,6 +259,14 @@ public void run() {
                     if (c != null) {
                         c.recordTermination();
                         c.disconnect(OfflineCause.create(hudson.model.Messages._Hudson_NodeBeingRemoved()));
+                        for(Executor executor: c.getAllExecutors()){
```

Comment:
LGTM, but dummy question, more for my understanding than an actual concern I think: shouldn't the interruption of executors be actually called *before* disconnecting the `Computer` on the previous line?